src/main/resources/application.conf

```diff
@@ -43,6 +43,9 @@ backend {
   //  baseExecutionBucket = ""
   //  endpointUrl = ""
   //}
+  local {
+    tryToHardlinkInputs = true
+  }
```

Comment:
I'm hesitant to plan for the existence of the sge backend (or any other backend) before it really exists although I think this is a good idea in general.

This probably should come in later though, there are some complexities here that need to be worked out. For instance I like that we're moving "cromwell-executions" from being hardwired to here, but the JES backend should know the same thing and it shouldn't be looking at local-filesystem.

All simple stuff but out of scope for what @kcibul needs